addons/website/models/ir_qweb.py

```diff
@@ -56,6 +57,7 @@ def _prepare_frontend_environment(self, values):
         """
         irQweb = super()._prepare_frontend_environment(values)
 
+        values['get_pure_domain'] = get_pure_domain
```

Comment:
@rdeodoo 

> Also, since if this was an util, it should be an Odoo util and not a website util, as it's not related to website.

Yes I can put the util in base or something like that but I'm not sure to understand your point. Domain name seems very related to website. Isn't it ? :thinking: 

Ok for the rest :+1:

Reply:
Wanting to extract the "base"/"pure"/"netloc" domain from an URL is not related to the website.
Anyone might need it for any given URL, that's what I meant, if it was an util to do so as you did, it should probably be in base and just take an url as param, and website calls it passing the website domain.
(Typical pattern for most of our util has always been the same: we create in website, another module needs it and move it to base).